blitz_cli_rest_netconf_gnmi/README.md

```diff
@@ -0,0 +1,18 @@
+# Overview
+
+This connects to your device via the following methods:
+* Cli
+* Rest
+* Netconf
+* Gnmi
+
```

Comment:
And execute 5 testcases

* 4 Triggers (one for each connection) will change the bgp neighbor 10.1.1.2 as-path, verify it changed, and change it back.

* 1 Trigger...

better formating